model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/schema.proto

```diff
@@ -139,6 +139,14 @@ message LogicalTypes {
     //     corresponds to chronological order.
     MILLIS_INSTANT = 2 [(org.apache.beam.model.pipeline.v1.beam_urn) =
       "beam:logical_type:millis_instant:v1"];
+
+    // A URN for Decimal type
+    //   - Representation type: BYTES
+    //   - A decimal number with specified scale and value. Its BYTES
```

Comment:
IIUC this type actually has _unspecified_ scale (while fixed_decimal has specified scale). Is that right?

Reply:
decimal has _unspecified_ precision and specified scale; while fixed_decimal has both precision and scale specified.

Reply:
Right, so this comment is incorrect?

Reply:
Ohh, I see. It's saying the actual value is the specified precision and a value. I would probably document it like this:

```suggestion
    //   - A decimal number with variable scale. Its BYTES
```

Reply:
ah yeah the comment in the proto is incorrect. I think I copy-pasted the comment for FixedPrecisionDecimal to here, thanks for catching it.